it/content/src/main/content/jcr_root/conf/core-components-it/settings/wcm/policies/.content.xml

```diff
@@ -12,7 +12,7 @@
                       jcr:primaryType="nt:unstructured"
                       jcr:title="formcontainer-policy"
                       sling:resourceType="wcm/core/components/policy/policy"
-                      components="[/apps/forms-components-examples/components/form/textinput,/apps/forms-components-examples/components/form/datepicker]">
+                      components="[/apps/forms-components-examples/components/form/textinput,/apps/forms-components-examples/components/form/datepicker,/libs/core/fd/components/form/footer/v1/footer]">
```

Comment:
Lets create a proxy component in apps and use the link there.

Reply:
this is done, in current push, made a component in apps